pygmt/helpers/decorators.py

```diff
@@ -82,6 +82,13 @@
             ``[g|p]``
             Force output grid to be gridline (g) or pixel (p) node registered.
             Default is gridline (g).""",
+    "t": """\
+        transparency : float
+            Set transparency level, in (0-100] percent range.
```

Comment:
```suggestion
            Set transparency level, in (0-100) percent range.
```

Actually a typo on upstream GMT too https://docs.generic-mapping-tools.org/6.1/gmt.html#t-full.

Reply:
100 is a valid value. Actually, even 0 is valid.

Reply:
Ah right, [square bracket notation](https://stackoverflow.com/questions/4396290/what-does-this-square-bracket-and-parenthesis-bracket-notation-mean-first1-last). In this case, use:
```suggestion
            Set transparency level, in [0-100] percent range.
```